.gitignore

```diff
@@ -206,9 +205,3 @@ tags
 [._]*.un~
 
 # End of https://www.toptal.com/developers/gitignore/api/python,macos,vim
-
-# Data
-*.csv
-*.data
-*.txt
-
```

Comment:
`nit`: since this will be public, it might be worth keeping this inplace in case we get a PR? 🤷 Don't want to push data (once the data we need is uploaded). Thoughts?

Reply:
Restored in [e3f380a](https://github.com/enthought/Numpy-Tutorial-SciPyConf-2023/pull/1/commits/e3f380a228409784ade9b267a90abc275145573d)